src/locale/locales/ca/messages.po

```diff
@@ -1210,19 +1210,19 @@ msgstr "per <0/>"
 
 #: src/screens/Signup/StepInfo/Policies.tsx:80
 #~ msgid "By creating an account you agree to the {els}."
-#~ msgstr "Creant el compte indiques que estàs d'acord amb {els}."
+#~ msgstr "Creant un compte indiques que estàs d'acord amb {els}."
 
 #: src/screens/Signup/StepInfo/Policies.tsx:81
 msgid "By creating an account you agree to the <0>Privacy Policy</0>."
-msgstr ""
+msgstr "Creant un compte indiques que estàs d'acord amb la <0>Política de privadesa</0>."
 
 #: src/screens/Signup/StepInfo/Policies.tsx:48
 msgid "By creating an account you agree to the <0>Terms of Service</0> and <1>Privacy Policy</1>."
-msgstr ""
+msgstr "Creant un compte indiques que estàs d'acord amb les <0>Condicions del servei/0> i amb la <0>Política de privadesa</0>."
```

Comment:
```suggestion
msgstr "Creant un compte indiques que estàs d'acord amb les <0>Condicions del servei</0> i amb la <0>Política de privadesa</0>."
```
A missing `<` here

Reply:
Thanks, I have fixed that in a new PR, this on was approved so fast
https://github.com/bluesky-social/social-app/pull/5735